#diff -Nurd -x'*~' gnome-desktop-2.12.1.orig/libgnome-desktop/gnome-desktop-item.c gnome-desktop-2.12.1/libgnome-desktop/gnome-desktop-item.c
#--- gnome-desktop-2.12.1.orig/libgnome-desktop/gnome-desktop-item.c	2005-08-21 14:55:17.000000000 -0400
#+++ gnome-desktop-2.12.1/libgnome-desktop/gnome-desktop-item.c	2005-11-02 16:26:48.000000000 -0500
#@@ -36,7 +36,7 @@
# #include <sys/types.h>
# #include <dirent.h>
# #include <unistd.h>
#-#include <time.h>
#+#include <sys/time.h>
# #include <string.h>
# #include <glib/gi18n-lib.h>
# #include <libgnome/gnome-util.h>
diff -Nurd -x'*~' gnome-desktop-2.12.1.orig/ltmain.sh gnome-desktop-2.12.1/ltmain.sh
--- gnome-desktop-2.12.1.orig/ltmain.sh	2005-09-18 13:14:19.000000000 -0400
+++ gnome-desktop-2.12.1/ltmain.sh	2005-11-02 16:27:34.000000000 -0500
@@ -5341,10 +5341,6 @@
 
 # Directory that this library needs to be installed in:
 libdir='$install_libdir'"
-	  if test "$installed" = no && test "$need_relink" = yes; then
-	    $echo >> $output "\
-relink_command=\"$relink_command\""
-	  fi
 	done
       fi
 
